packages/ui/src/composites/wui-account-button/index.ts

```diff
@@ -26,17 +26,19 @@ export class WuiAccountButton extends LitElement {
 
   // -- Render -------------------------------------------- //
   public override render() {
+    const isEns = this.address.endsWith('.eth')
```

Comment:
This check and `isEns` variable are specific to ethereum, where as within ui / core and scafold we should remain chain agnostic. Instead, check for existance / length of `AccountController.state.profileName`